### DISCUSSION TITLE
Flutter Splash screen : for dark theme

### DISCUSSION THREAD

**Discussion by miadona** [community/community#29946] [Mobile] [answered]
Hell Team, I try implementing Splash screen but it works only for native with light theme.
When users change to dark theme, splash screen UI does match up, I trying to find solution where I would implement splash based on detecting theme use in mobile users

**Comment by surajadkhari** [comment, marked as answer]
I would suggest :
flutter_native_splash: 
https://pub.dev/packages/flutter_native_splash
You can define logo and colour for both light and dark themes:
![image](https://user-images.githubusercontent.com/48079501/185421958-c83b1a40-00dc-4fe3-9c50-0184636aea8d.png)

**Reply by miadona** [reply]
Finally, it works. Thank you
